frontend/src/v5/ui/components/viewer/cards/cardFilters/filterForm/filterFormValues/filterFormValues.component.tsx

```diff
@@ -94,6 +106,13 @@ export const FilterFormValues = ({ type }: { type: CardFilterType }) => {
 			</>
 		);
 	}
+	if (isSelectType(type)) {
+		return (
+			<FormSelect name="values.0.value" formError={!!error?.[0]}>
```

Comment:
Not a big deal, but can you use string interpolation like this? `${name}.0.value`